content/guides/building/get-source-git.md

```diff
@@ -137,6 +148,8 @@ git clone http://github.com/haiku/haiku.git
 
 Please note that these repositories do not contain any `hrev` tags, which are used by the Haiku build system to determine the Haiku revision. To work around this limitation, use the <a href="https://cgit.haiku-os.org/haiku/tree/build/jam/UserBuildConfig.ReadMe" target="_blank">HAIKU_REVISION build variable</a> when building Haiku.
 
+The limitation is caused by the fact that GitHub does not support repositories with tens of thousands of tags very well. A new tag is created once every few days and used to create our <a href="https://download.haiku-os.org">Nightly images</a>.
```

Comment:
This is incorrect. Tags are created whenever something is pushed to the master branch, not every few days (it's closer to multiple times a day). And "every few days" wouldn't make sense for nightly images.

Anyway, this kind of technical details should probably not be included in this page. Knowing that it won't work and how to fix/workaround it is enough here. Technical details should go somewhere else?

Reply:
> This is incorrect. Tags are created whenever something is pushed to the master branch, not every few days (it's closer to multiple times a day). And "every few days" wouldn't make sense for nightly images.

I remembered a conversation we had because of this subject, I'd definitely go "huh, why" and there's no other designated place for that short detail.

Reply:
The answer to "why" is mainly "stupid historical reasons when we switched from SVN".

We could use "git describe --tags" or similar and our life would be a lot simpler. But back when we switched to git, there was no such thing or it was too slow to be usable.

But there could be a page or two of text explaining how the hrevs (and btrevs) work, how they are generated, why they are needed, what problems they create, etc. Then it's not "just a short detail" anymore AND we can provide accurate info :)

Reply:
*misremembered, my bad, I just started contributing again. :D